examples/LLM/llama2/chat_app/docker/torchserve_server_app.py

```diff
@@ -0,0 +1,193 @@
+import json
+import os
+import time
+
+import requests
+import streamlit as st
+
+MODEL_NAME = os.environ["MODEL_NAME"]
+MODEL = MODEL_NAME.split("---")[1]
+
+# App title
+st.set_page_config(page_title="TorchServe Server")
+
+
+def start_server():
+    os.system("torchserve --start --ts-config /home/model-server/config.properties")
```

Comment:
This would show as success even if the the server failed to start, favor subprocess instead, check the return code and then query torchserve directly to see server health as opposed to using sleep

Reply:
Oh..good point..let me try. This command was returning immediately. I did try with ping but that was failing as the server was not up yet.

Reply:
I changed the logic, but it still doesn't work as expected. There is a slight difference between when the command returns and when the server starts. Add a check with ping , but needs a sleep.